src/AsmResolver.PE/Debug/CodeView/CodeViewDataSegment.cs

```diff
@@ -23,16 +23,20 @@ public abstract class CodeViewDataSegment : SegmentBase, IDebugDataSegment
         /// Creates a new CodeViewDataSegment depending on CodeView Signature
         /// </summary>
         /// <param name="reader">The input stream to read from.</param>
+        /// <param name="context">Context for the reader</param>
         /// <returns></returns>
-        public static CodeViewDataSegment FromReader(IBinaryStreamReader reader)
+        public static CodeViewDataSegment FromReader(IBinaryStreamReader reader, PEReaderContext context)
```

Comment:
Reverse parameter order to conform with the style of the rest of AsmResolver's API for consistency.